src/dsp_tools/models/helpers.py

```diff
@@ -25,11 +24,7 @@ class OntoIri:
     hashtag: bool
 
 
-ContextType = NewType("ContextType", dict[str, OntoIri])
-
-
-def LINE() -> int:
-    return sys._getframe(1).f_lineno
+ContextType = dict[str, OntoIri]
```

Comment:
- pylint doesn't support NewType (see https://github.com/pylint-dev/pylint/issues/3162#issuecomment-539970727), so I went for a type alias instead.
- `LINE()` is nowhere used in the codebase, so I can get rid of this protected-member-access (`sys._getframe`)